src/main/java/com/iota/iri/zmq/MessageQ.java

```diff
@@ -26,6 +44,14 @@ public static MessageQ createWith(ZMQConfig config) {
         return new MessageQ(config.getZmqPort(), config.getZmqIpc(), config.getZmqThreads(), config.isZmqEnabled());
     }
 
+    /**
+     * Creates and starts a ZMQ publisher.
+     *
+     * @param port port the publisher will be bound to
+     * @param ipc IPC socket to bind the publisher to
```

Comment:
```suggestion
     * @param ipc IPC socket the publisher will be bound to
```
or swap the `port` doc to this